fbc/4.12/openshift-builds-operator/catalog.yaml

```
@@ -52,249 +64,377 @@ name: openshift-builds-1.0
 package: openshift-builds-operator
 schema: olm.channel
 ---
+schema: olm.deprecations
+package: openshift-builds-operator
+entries:
+  - reference:
+      schema: olm.bundle
+      name: openshift-builds-operator.v1.0.0
+    message: |
+      openshift-builds-operator.v1.0.0 is deprecated. Uninstall and install openshift-builds-operator.v1.1.0 or later.
+  - reference:
+      schema: olm.bundle
+      name: openshift-builds-operator.v1.0.1
+    message: |
+      openshift-builds-operator.v1.0.1 is deprecated. Uninstall and install openshift-builds-operator.v1.1.0 or later.
+  - reference:
+      schema: olm.bundle
+      name: openshift-builds-operator.v1.0.2
+    message: |
+      openshift-builds-operator.v1.0.2 is deprecated. Uninstall and install openshift-builds-operator.v1.1.0 or later.
+  - reference:
+      schema: olm.channel
+      name: stable-v1
+    message: |
+      channel 'stable-v1' is deprecated. Please switch to channel 'latest', 'builds-1.1' or later.
+  - reference:
+      schema: olm.channel
+      name: openshift-builds-1.0
+    message: |
+      channel 'openshift-builds-1.0' is deprecated. Please switch to channel 'latest', 'builds-1.1' or later.
```

This is more of an "end of life" notice:

```suggestion
      channel 'openshift-builds-1.0' is deprecated and will reach end of maintenance support soon. Please switch to channel 'latest', 'builds-1.1' or later.
```